pkg/sink/mysql/config_test.go

```diff
@@ -245,6 +246,11 @@ func TestParseSinkURIOverride(t *testing.T) {
 		checker: func(sp *Config) {
 			require.EqualValues(t, sp.tidbTxnMode, defaultTiDBTxnMode)
 		},
+	}, {
+		uri: "mysql://127.0.0.1:3306/?prep-stmt-cache-size=1000000", // int32 max
```

Comment:
Does it mean to test a size that is larger than `maxPrepStmtCacheSize`?

Reply:
yes, will correct it